p2p/src/main/kotlin/maru/p2p/MaruPeer.kt

```diff
@@ -72,15 +88,41 @@ class DefaultMaruPeer(
       val statusMessage = statusMessageFactory.createStatusMessage()
       val sendRpcMessage: SafeFuture<Message<Status, RpcMessageType>> =
         sendRpcMessage(statusMessage, rpcMethods.status())
```

Comment:
I think we should handle failures here, during `sendRpcMessage` call. Because if we have a network disconnect, we'll fail to send the message. This call will fail and the next status update call won't ever be scheduled

Reply:
If we have a network disconnect we WANT to disconnect the peer, as it is useless.

Reply:
Let me clarify. If there's a network disruption for 100 milliseconds, all the peers will be lost immediately? There's 0 grace period?

Reply:
We could retry to send the message, but I'm not sure it is worth it. We do have many peers and we will find another peer to fill the gap.